### PR TITLE
Accepted ciphers

### PR DESCRIPTION
**Pull request by buckley-w-david**
Redo of #116 because it got closed when I deleted the base branch and now I can't re-open it.

### CODE REVIEW

**Comment by buckley-w-david** [comment]
pinging @obrien-j because it's a new PR

File: tracker/data/processing.py [hunk 1]
```diff
@@ -36,6 +36,45 @@
 SCAN_CACHE = os.path.join(env.SCAN_DATA, "cache")
 SCAN_DOMAINS_CSV = os.path.join(SCAN_CACHE, "domains.csv")
 
+ACCEPTED_CIPHERS = {
```
Comment (obrien-j) [inline tracker/data/processing.py]: @poteaduncan @buckley-w-david It'd be super nice if these were hosted by CSE or TBS **NOT** in a PDF form, such that an update to the cipher list would be able to easily allow this service to auto-update it's scan list without having to push a code-update.

Still don't like this in code, would rather this be an externally managed file. (still merge it though)

File: tracker/data/processing.py [hunk 2]
```diff
@@ -579,16 +619,17 @@ def https_behavior_for(name, pshtt, sslyze, parent_preloaded=None):
         # ITPIN cares about usage of TLS 1.0 and TLS 1.1
         tlsv10 = boolean_for(sslyze["TLSv1.0"])
         tlsv11 = boolean_for(sslyze["TLSv1.1"])
+        uses_accepted_ciphers = all(cipher in ACCEPTED_CIPHERS for cipher in sslyze.get("Accepted Ciphers").split(', '))
```
Comment (obrien-j) [inline tracker/data/processing.py]: Are you going to pull out the ciphers that the service is using that we don't support? or did that get fixed up somewhere else?

File: tracker/data/processing.py (hunk 1 above)
Comment (ptd-tbs) [reply tracker/data/processing.py]: I agree - it should be kept in a separate file similar to how the domain list is kept separate.

**Pull request merged**